pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -231,13 +232,19 @@ private static void start() {
         int requestThreadTimeout = cfg.getInt(ConfigFile.THREAD_TIMEOUT);
         String healthcheckPassword = cfg.getString(ConfigFile.HEALTHCHECK_PASSWORD);
 
-        int port = cfg.getInt(ConfigFile.PORT);
-        port(port);
+        // app engine's port env var wins
+        int configFilePort = cfg.getInt(ConfigFile.PORT);
+        String appEnginePort = System.getenv(APP_ENGINE_PORT);
+        if (appEnginePort != null) {
+            port(Integer.parseInt(appEnginePort));
+        } else {
+            port(configFilePort);
+        }
 
```

Comment:
@zyme One change that you had made in PR in old repo : that I did not carry forward is photographed below. 
https://github.com/broadinstitute/ddp/pull/1160/files#r346982149 
Was unclear to me what the reason for it was. Let me know:

![image](https://user-images.githubusercontent.com/29984380/73374514-53f87a00-4288-11ea-9417-b2c97266f422.png)

Reply:
I can't recall what that hurdle was, but let's keep that change out of this PR as you suggest.